CloudCulator/Cloudculator.Core.UnitTests/CalculatorEngineUnitTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Cloudculator.Core.UnitTests
+{
+    public class CalculatorEngineUnitTests
+    {
+        [Fact]
+        public void Add_TwoNumbers_ReturnAddition()
+        {
+            //Arrange
+            var service = new CalculatorEngine();
```

Comment:
This part of setup can be set once outside the methods

Reply:
Done